portfolio/src/main/webapp/script.js

```diff
@@ -64,23 +64,23 @@ function createListElement(key, text) {
   liElement.innerText = text.comment + ',' + text.userEmail + '\t';
 
   // Create an input element to delete item.
-  var form = document.createElement('form');
+  const form = document.createElement('form');
   form.setAttribute('method', 'post');
   form.setAttribute('action', '/delete-data');
 
   // Create hidden element to store id of button.
-  var id = document.createElement('input');
+  const id = document.createElement('input');
```

Comment:
id is a very generic term, what is this id supposed to identify?

Reply:
replaced with comment_id, (it's the unique key from the database used to identify the comment)